form.yml

```diff
@@ -0,0 +1,326 @@
+  sapcar_exe_file:
+    $name: Path to sapcar executable if extracting NW SAR archive
+    $visibleIf: .use_swpm_sar_file == true
+    $type: text
+    $help: The path to sapcar executable to extract NW SAR archive
+    $optional: true
+
+  swpm_folder:
+    $name: SWPM folder location
+    $type: text
+    $help: Path to already extracted SWPM installer folder
+    $optional: true
+  
+  sapexe_folder:
+    $name: SAP exe folder location
+    $type: text
+    $help: Path to sap exe folder
+    $optional: false
+  
+  installation_folder:
+    $name: Netweaver installation folder
+    $type: text
+    $default: /tmp/swpm_unattended
+    $help: Path to run the installation
+    $optional: false
+  
+  ha_enabled:
+    $name: Enable HA cluster configuration
+    $type: boolean
+    $default: true
+    $help: Enable the HA cluster configuration
+    $optional: false
+
+  nfs_version:
+    $name: NFS version
+    $type: text
+    $default: nfs4
+    $help: Used to connect to the nfs share
+    $optional: true
+
+  nfs_options:
+    $name: NFS options
+    $type: text
+    $default: defaults
+    $help: Specify the options for nfs share
+    $optional: true
+
+  additional_dvds:
+    $name: Additional dvds
+    $type: edit-group
+    $help: Path to software folder or archives with additional SAP software
+    $minItems: 1
+    $prototype:
+        $name: Additional DVD
+        $type: text
+        $optional: false
+
+  saptune_solution:
+    $name: saptune solution to apply
+    $type: text
+    $default: NETWEAVER
+    $help: saptune solution to apply to all nodes
+    $optional: true
+
+  hana_checkbox:
+    $name: Add information about installed HANA database (mandatory for DB/PAS/AAS instances)
+    $type: boolean
+    $default: false
+    $help: Information about the already deployed HANA Database. It is mandatory if DB/PAS/AAS are to be installed
+  hana:
+    $name: Hana database information
+    $visibleIf: .hana_checkbox == true
+    $type: group
+    $help: Information about the already deployed HANA Database. Mandatory for DB/PAS/AAS instances
+    host:
+      $name: Host
+      $type: text
+    sid:
+      $name: SID
+      $type: text
+    instance:
+      $name: Instance
+      $type: text
+    password:
+      $name: Password
+      $type: password
+
+  schema_checkbox:
+    $name: Add schema information from DB installation
+    $type: boolean
+    $default: false
+    $help: Information about the Schema created during DB installation and used by PAS/AAS
+  schema:
+    $name: Schema
+    $visibleIf: .schema_checkbox == true
+    $type: group
+    $help: Information about the Schema created during DB installation and used by PAS/AAS
+    name:
+      $name: Schema name
+      $type: text
+    password:
+      $name: Password
+      $type: password
+
+  product_id:
+    $name: SAP product ID
+    $type: text
+    $help: The installed SAP product id. If some particular node has its own product_id that one will have the preference
```

Comment:
We need to add more information here. The product ID is not something everyone will know (actually I would be nobody knows, the options are. `NW750.HDB.ABAPHA, S4HANA1709.CORE.HDB.ABAPHA, S4HANA1809.CORE.HDB.ABAPHA, S4HANA1909.CORE.HDB.ABAPHA`. If we remove the `HA` from the end, it means a non HA environment (without ERS)